classes/game.rb

```diff
@@ -0,0 +1,23 @@
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(last_played_at, multiplayer)
+    super()
```

Comment:
you can consider the following code snippet 
```suggestion
    
```
 def initialize(last_played_at, multiplayer, publish_date)
    super(publish_date)
    @id = Random.rand(1..1000)
    @last_played_at = last_played_at
    @multiplayer = multiplayer
  end
so it would be great to follow the project requirements 
![gg](https://github.com/AbdusSattar-70/Ruby-Group-Capstone/assets/118761303/51d6a767-e2b2-4794-bb2e-4b0105fe9dc6)